Core/src/main/java/com/plotsquared/core/util/query/OwnersIncludeFilter.java

```diff
@@ -23,17 +23,11 @@
 
 import java.util.UUID;
 
-class OwnersIncludeFilter implements PlotFilter {
-
-    private final UUID owner;
-
-    OwnersIncludeFilter(final @NonNull UUID owner) {
-        this.owner = owner;
-    }
+record OwnersIncludeFilter(UUID owner) implements PlotFilter {
 
     @Override
     public boolean accepts(final @NonNull Plot plot) {
-        return plot.isBasePlot() && plot.getOwners().size() > 0 && plot.getOwners().contains(owner);
+        return plot.isBasePlot() && plot.isOwner(owner);
```

Comment:
the current `Plot#getOwners#contains` takes server-plots into account (so, if the server uuid is passed into this filter, and a plot is server owned, this filter would return true). `Plot#isOwner` uses `Plot#getOwnerAbs` and therefor would not produce the same result. Haven't seen an occurrence of a server plot being queried - not sure if this should just be changed either tbh